src/messages.rs

```diff
@@ -46,7 +46,7 @@ pub fn client_connection_preamble(id: ClientId) -> Rc<MallocMessageBuilder> {
 pub fn append_entries_request(term: Term,
                               prev_log_index: LogIndex,
                               prev_log_term: Term,
-                              entries: &[(Term, &[u8])],
+                              entries: &[(Term, Vec<u8>)],
```

Comment:
I think changing this back to `&[u8]` would allow you to get rid of the `to_owned`, with no loss in functionality, right?